index.html

```diff
@@ -2,11 +2,183 @@
+          </dd>
+          <img src="https://via.placeholder.com/300x250" alt="фото косметики">
+        </dl>
+        <dl>
+          <dt>Парфюмерия</dt>
+          <dd>Оригинальная нишевая парфюмерия известных брендов напрямую от производителей доступна вам в России. Наши духи и ароматы 100% оригинальны, что можно проверить по кодам на каждом флаконе.
+              Мы предлагаем вам более 40 брендов парфюмерии, среди которых: BOIS, CIRA, ALTAIA, Fragonard, Rosendo Mateu, Olfattology, Robert Piguet, Eau D’Italie, LINARI, Nobile 1942, Guido Crepax Valentina, Simone Anderoli и другие.
+          </dd>
+          <img src="https://via.placeholder.com/300x250" alt="фото парфюмов">
+        </dl>
+      </section>
+      <section>
+        <h2 class="visually-hidden">Список приемуществ</h2>
+        <ul>
+          <li>
+            <p>150</p>
+            <p>Партнеров-магазинов по всей России</p>
+          </li>
+          <li>
+            <p>500</p>
+            <p>Салонов красоты наши клиенты</p>
+          </li>
+          <li>
+            <p>49</p>
+            <p>Брендов доступны для заказа</p>
+          </li>
+        </ul>
+      </section>
+      <section>
+        <h2>Бренды</h2>
+        <ul>
+          <li>Brazzcare</li>
+          <li>GreenLight</li>
+          <li>That’so</li>
+          <li>Ellipse</li>
+          <li>SO UP RELOADED</li>
+        </ul>
+        <ul>
+          <li>Ameia</li>
+          <li>Biogenie Face & Buste</li>
+          <li>Venusian CO2</li>
+          <li>Plasmage</li>
+          <li>Skin Master Plus</li>
+        </ul>
+        <ul>
+          <li>BOIS</li>
+          <li>CIRA</li>
+          <li>ALTAIA</li>
+          <li>Fragonard</li>
+          <li>Rosendo Mateu</li>
+          <li>Olfattology</li>
+        </ul>
+        <ul>
+          <li>Eau D'Italie</li>
+          <li>LINARI</li>
+          <li>Nobile 1942</li>
+          <li>Guido Crepax Valentina</li>
+          <li>Simone Andreoli</li>
+        </ul>
+      </section>
+    </main>
+    <footer class="main-footer">
+      <section>
+        <h2>Контакты компании</h2>
+        <p>
+          <a>phone icon</a><a href="tel:84951092313">8 (495) 109-23-13</a> доб.103, 104, 105;<br>
+          <a href="tel:89256499834">8 (925) 649-98-34</a><br>
+          <a>mail icon</a><a href="mailto:zakaz@solingcompany.ru">zakaz@solingcompany.ru</a><br>
+          <a href="mailto:parfummanager@linkparfum.ru">parfummanager@linkparfum.ru</a>
+          <a>map icon</a>Москва, ул. Павловская, дом 27 стр 3<br>
+          <a>insta icon</a> @amiea_russia; @brazzcare_russia; @greenlight_russia; @solingparfums
+        </p>
+      </section>
+      <section>
+        <h2>Юридические данные</h2>
+        <p>
+          ООО «Солинг»<br>
+          ИНН 6722027595<br>ОГРН 1126722000591
+        </p>
+        <a href="#">Заказать звонок</a>
+      </section>
+      <p>2022  @ all right reserved</p>
+    </footer>
```

Comment:
You have "thanks pop-up" block at the mock-up. 
Usually we place pop-ups after the footer section. 
Don't forget the mark-up here.